english/stairs/stairs.md

```diff
@@ -0,0 +1 @@
+You race for the twisting stair case. The officer is catching up, but you race up the stairs. As you round your way up the stairs you hear the officer pounding up behind you. You glance back and see him only a few steps away!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.